p9Layouts/defaultPnorLayout_64.xml

```diff
@@ -386,4 +386,14 @@ Layout Description
         <sha512Version/>
         <readOnly/>
     </section>
+    <section>
+        <description>Hostboot Runtime Proxy (32KB)</description>
+        <eyeCatch>HBRT_PROXY</eyeCatch>
+        <physicalOffset>0x36A9000</physicalOffset>
+        <physicalRegionSize>0x8000</physicalRegionSize>
+        <side>A</side>
```

Comment:
would have picked sideless of consistency but the current claim is that nothing cares about this.